backend/src/types/advisor.ts

```
@@ -0,0 +1,57 @@
+  description: string;
+  affectedObject?: string;
+  recommendation?: string;
+}
+
+/**
+ * Suppression scope and reason. The `as const` arrays are the single source of
+ * truth: route validation iterates them and the union types are derived from
+ * them, so adding/removing a value updates both at once.
+ */
+export const SUPPRESSION_SCOPES = ['instance', 'rule'] as const;
+export const SUPPRESSION_REASONS = ['false_positive', 'accepted_risk', 'wont_fix', 'other'] as const;
```

ESLint

[error] Replace `'false_positive',·'accepted_risk',·'wont_fix',·'other'` with `⏎··'false_positive',⏎··'accepted_risk',⏎··'wont_fix',⏎··'other',⏎`

(prettier/prettier)